cosmovisor/scanner.go

```diff
@@ -158,9 +158,5 @@ func parseUpgradeInfoFile(filename string) (upgradetypes.Plan, error) {
 		return upgradetypes.Plan{}, fmt.Errorf("invalid upgrade-info.json content; name and height must be not empty; got: %v", ui)
 	}
 
-	// Normalize name to prevent operator error in upgrade name case sensitivity
-	// errors.
-	ui.Name = strings.ToLower(ui.Name)
```

Comment:
This was added in https://github.com/cosmos/cosmos-sdk/pull/11608, but I believe only the `EqualFold` addition was necessary. Otherwise, we get #12915 bug.

Reply:
So was not a  bug, only not documented.

Reply:
@alexanderbez should I revert that change and add docs?

Reply:
@julienrbrt yes pls.

Reply:
Sure!

Reply:
Done in #12921